tests/openvino/test_diffusion.py

```diff
@@ -77,8 +77,8 @@ class OVPipelineForText2ImageTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES = ["stable-diffusion", "stable-diffusion-xl", "latent-consistency"]
     NEGATIVE_PROMPT_SUPPORT_ARCHITECTURES = ["stable-diffusion", "stable-diffusion-xl", "latent-consistency"]
```

Comment:
now that `variant` was added, it could make sense to add a test to make sure this works as expected / doesn't break in the future, can be added in a following PR

Reply:
added in https://github.com/huggingface/optimum-intel/pull/1106/commits/b27e6f79743e273578eead971126d88c2cf6d297 thanks a lot @eaidova !